js/src/forum/components/Notification.js

```diff
@@ -8,26 +8,25 @@ import Button from '../../common/components/Button';
  * The `Notification` component abstract displays a single notification.
  * Subclasses should implement the `icon`, `href`, and `content` methods.
  *
- * ### Props
+ * ### Attrs
  *
  * - `notification`
  *
  * @abstract
  */
 export default class Notification extends Component {
   view() {
-    const notification = this.props.notification;
+    const notification = this.attrs.notification;
     const href = this.href();
 
+    const linkAttrs = {};
+    linkAttrs[href.indexOf('://') === -1 ? 'route' : 'href'] = href;
+
     return (
       <a
         className={'Notification Notification--' + notification.contentType() + ' ' + (!notification.isRead() ? 'unread' : '')}
-        href={href}
-        config={function (element, isInitialized) {
-          if (href.indexOf('://') === -1) m.route.apply(this, arguments);
-
-          if (!isInitialized) $(element).click(this.markAsRead.bind(this));
-        }}
+        {...linkAttrs}
+        onclick={this.markAsRead.bind(this)}
```

Comment:
Not sure. I think the performance impact is minimal, so that shouldn't matter? We could move it to an oncreate, but I don't see why that was needed in the first place either.

Reply:
Or a readability one. Either way, this is cleaner.